src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -7,24 +7,70 @@
 public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
-
+    public String command= new String();
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public HelpCommand() {}
+    public HelpCommand(String command) {
+        this.command = command;
+
+    }
 
     @Override
     public CommandResult execute() {
-        return new CommandResult(
-                AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
-        );
+        switch(command.toLowerCase().trim()){
```

Comment:
Many violations of our coding standard for `switch` statement (e.g. empty spaces, indentation), check out our coding standard.